Tasks/AndroidSigning/task.json

```diff
@@ -91,6 +95,7 @@
         },
         {
             "name": "jarsignerArguments",
+            "aliases": ["jarsignArguments"],
```

Comment:
`jarsign` should be `jarsigner` to match the name of the tool.

Reply:
So, you'd like the two places above to be `jarsignerKeyPassword` and `jarsignerKeystoreAlias`, and remove this line?